pkg/harvester/l10n/en-us.yaml

```
@@ -594,6 +595,10 @@ harvester:
       userTips: The user to be added must already exist; otherwise, the credentials will not take effect.
       duplicatedUser: User already exists.
       invalidUser: Invalid Username.
+    lockIconTooltip:
+      image: Image encrypted
+      volume: Volume encrypted
+      both: Image encrypted and volume encrypted
```

Others LGTM. Just one issue, It should be:

```suggestion
      image: Encrypted Image
      volume: Encrypted Volume 
      both: Encrypted Image and Encrypted Volume 
```

---

Updated in  52702fb